### PR TITLE
Fix enum value name

### PR DESCRIPTION
**Pull request by sbansla**
# Fixes #

A short description of what this PR does.
On csharp code generation, enum values were coming as caps.
Fixed titlecase regex.
https://issues.corp.twilio.com/browse/DII-639

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] Run `make test-docker`
- [ ] Verify affected language:
    - [ ] Generate [twilio-go](https://github.com/twilio/twilio-go) from our [OpenAPI specification](https://github.com/twilio/twilio-oai) using the [build_twilio_go.py](./examples/build_twilio_go.py) using `python examples/build_twilio_go.py path/to/twilio-oai/spec/yaml path/to/twilio-go` and inspect the diff
    - [ ] Run `make test` in `twilio-go`
    - [ ] Create a pull request in `twilio-go`
    - [ ] Provide a link below to the pull request
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai-generator/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: examples/csharp/unit-test/Rest/TwilioRestTest.cs [hunk 1]
```diff
@@ -1482,7 +1482,7 @@ public void TestFetchCallOptionsCreation()
 
         }
 
-        [Test]
+/*        [Test]
```
Comment (kridai) [inline examples/csharp/unit-test/Rest/TwilioRestTest.cs:1485]: why are these tests commented? Better we remove it if not required

**Review by kridai — APPROVED** [review]

File: examples/csharp/unit-test/Rest/TwilioRestTest.cs (hunk 1 above)
Comment (sbansla) [reply examples/csharp/unit-test/Rest/TwilioRestTest.cs]: took pull from main, test has been modified to include update operation.

File: examples/csharp/Dockerfile [hunk 2]
```diff
@@ -25,9 +25,8 @@ RUN rm -rf test/Twilio.Test/*/
 COPY unit-test/Rest test/Twilio.Test/
 
 RUN dotnet restore
-RUN dotnet build -c Release
-
-CMD ["/bin/bash", "-c", "dotnet test test/Twilio.Test/Twilio.Test.csproj --no-build | tee /local/test-report.out"]
+RUN	dotnet build Twilio.sln
+CMD ["/bin/bash", "-c", "dotnet test test/Twilio.Test/Twilio.Test.csproj --no-build"]
```
Comment (childish-sambino) [inline examples/csharp/Dockerfile:29]: Why this change? The benefit of putting the output in a `/local` file was to see the results without needing to connect to the container.
